test/e2e/data/infrastructure-docker/v1alpha4/bases/cluster-with-kcp.yaml

```diff
@@ -62,14 +62,21 @@ spec:
         extraArgs: {enable-hostpath-provisioner: 'true'}
       apiServer:
         # host.docker.internal is required by kubetest when running on MacOS because of the way ports are proxied.
-        certSANs: [localhost, 127.0.0.1, 0.0.0.0, host.docker.internal]
+        certSANs: [localhost, 127.0.0.1, 0.0.0.0, "::", "::1", host.docker.internal]
     initConfiguration:
+      localAPIEndpoint:
+        advertiseAddress: '::'
```

Comment:
Assuming that we will run tests both in ipv4 and ipv6 modes, this change could have some tricky impacts...

We should think if there is a way to avoid us to duplicate templates, and if this is not possible and more variants of the templates will exists, we should think how to make the tests aware of which test to use...

Reply:
resolving because I have summarised consideration about ipv6 E2E testing down in a comment